packages/app/src/domain/vaccine/vaccine-page-introduction.tsx

```diff
@@ -125,21 +139,59 @@ export function VaccinePageIntroduction({
         </Box>
       </Tile>
 
-      <Box px={{ _: 3, sm: 4 }}>
-        <ContentHeader
-          subtitle={text.description}
-          reference={text.reference}
-          metadata={{
-            datumsText: text.datums,
-            dateOrRange: data.vaccine_administered_total.last_value.date_unix,
-            dateOfInsertionUnix:
-              data.vaccine_administered_total.last_value.date_of_insertion_unix,
-            dataSources: [],
-            moreInformationLabel: text.more_information.label,
-            moreInformationLink: text.more_information.link,
-          }}
-        />
-      </Box>
+      <TwoKpiSection>
+        <Box>
+          <Heading level={3}>{pageInfo.title}</Heading>
+          <Tile>
+            <RichContent blocks={pageInfo.description} />
+            <Box spacing={{ _: 3, md: 0 }} display="flex">
+              <MetadataBox>
+                <Metadata
+                  datumsText={text.datums}
+                  dateOrRange={
+                    data.vaccine_administered_total.last_value.date_unix
+                  }
+                  dateOfInsertionUnix={
+                    data.vaccine_administered_total.last_value
+                      .date_of_insertion_unix
+                  }
+                  dataSources={[]}
+                  moreInformationLabel={text.more_information.label}
+                  moreInformationLink={text.more_information.link}
+                  accessibilitySubject={''}
+                />
+              </MetadataBox>
+            </Box>
+          </Tile>
+        </Box>
+        <Box>
+          <Heading level={3}>{pageLinksTitle}</Heading>
+          <DecoratedLinksTile>
+            <Box>
+              {pageLinks.map((x, index) => (
+                <DecoratedLink link={x} compact={index > 0} key={x.href} />
+              ))}
+            </Box>
+          </DecoratedLinksTile>
+        </Box>
+      </TwoKpiSection>
     </Box>
   );
 }
+
+const MetadataBox = styled(Box)(
```

Comment:
let's try to step away from enhancing the Box component (and in this case it also isn't necessary because no props were passed on the MetadataBox component)
```suggestion
const MetadataBox = styled.div(
```

Reply:
Oh yea, this was a copy/paste action. Good point.